File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L3/09/Level3.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function main() {
  let vol: number = 0;
  let luasAlas: number = 0;

  console.log("Enter the radius and length of a cylinder: ");
  rl.question("Radius: ", (radiusStr) => {
    const radius: number = parseFloat(radiusStr);
    rl.question("Length: ", (lengthStr) => {
      const length: number = parseFloat(lengthStr);

      luasAlas = radius * radius * 3.14159;
      vol = luasAlas * length;

      console.log("The area is " + luasAlas);
      console.log("The volume of the cylinder is " + vol);

      rl.close();
    });
  });
}

main();

File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/plagiarized/L3/09/Level3.py
import math

radius = float(input("Enter the radius and length of a cylinder: "))
length = float(input())

luas_alas = radius ** 2 * math.pi
vol = luas_alas * length

print(f"The area is {luas_alas:.5f}")
print(f"The volume of the cylinder is {vol:.5f}")

